### PR TITLE
release(main):  v0.18.0

### PR DESCRIPTION
**Pull request by dkoshkin**
:robot: I have created a release *beep* *boop*
---

### Exciting New Features 🎉
* feat: Add feature-gates plumbing by @jimmidyson in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/919
* feat(api): Add kubernetes version to coredns version mapping by @thunderboltsid in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/939
### Fixes 🔧
* fix: Shorten readiness probe period to try to prevent races by @jimmidyson in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/930
* fix: Rename webhook container to manager by @jimmidyson in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/932
* fix: list correct registry.k8s.io/sig-storage/csi-snapshotter image by @dkoshkin in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/943
* fix: include kube-vip image in generated caren-images.txt by @dkoshkin in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/940
### Other Changes
* test(e2e): Add v1.30.5 test for Nutanix by @thunderboltsid in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/931
* build: Enable building binary only on macos by @jimmidyson in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/918
* build(deps): Update clusterctl binary to v1.8.3 by @thunderboltsid in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/929

**Full Changelog**: https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/compare/v0.17.0...v0.18.0

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Review by supershal — APPROVED** [review]

**Review by jimmidyson — APPROVED** [review]

**Review by supershal — APPROVED** [review]

File: test/e2e/data/shared/v1beta1-caren/metadata.yaml [hunk 1]
```diff
@@ -43,3 +43,6 @@ releaseSeries:
   - contract: v1beta1
     major: 0
     minor: 18
+  - contract: v1beta1
+    major: 0
+    minor: 19
```
Comment (jimmidyson) [inline test/e2e/data/shared/v1beta1-caren/metadata.yaml:48]: If we're not bumping to v0.19.99 in caren.yaml then we don't need this here.

Comment (dkoshkin) [reply test/e2e/data/shared/v1beta1-caren/metadata.yaml]: Removed.

File: test/e2e/config/caren.yaml [hunk 2]
```diff
@@ -136,7 +136,7 @@ providers:
   versions:
   # Upgrade e2e tests will use this as the "upgrade from" version.
   # This should reference the most recent successful release.
-  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.15}"
+  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.17}"
     value: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/download/{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.15}/runtime-extension-components.yaml"
```
Comment (jimmidyson) [inline test/e2e/config/caren.yaml:140]:
```suggestion
    value: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/download/{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.17}/runtime-extension-components.yaml"
```

Comment (dkoshkin) [reply test/e2e/config/caren.yaml]: Thanks, fixed.

**Review by faiq — APPROVED** [review]

**Review by faiq — APPROVED** [review]

**Pull request merged**